### PR TITLE
updated UX for videos section, now have responsive css slider

### PR DESCRIPTION
**Pull request by kkemple**
addresses issue #96 
I updated the slider section to be a responsive css3 slider, also made it much more apparent that there are multiple videos, see screenshots below

> NOTE: the arrows for the slider nav are .png files, I know @paulfalgout has done a lot of work with the svg changes so this will need that treatment as well, but its a good start

slider-1

slider-2

slider-3

slider-in-action

### CODE REVIEW

**Comment by kkemple** [comment]
third image is when slider is below minimum breakpoint vid is 100%

**Comment by kkemple** [comment]
also, i hid the big block of text (figcaption), i wasn't sure how that would play in to new design, i don't feel we need it with the new design, I can remove (or make title or something) and update PR if needed

**Comment by paulfalgout** [comment]
I :heart: this.  
:wave: bye-bye to jquery-cycle.

**Comment by paulfalgout** [comment]
So the two biggest things for the merge are likely going to be:
https://github.com/marionettejs/marionettejs.com/pull/101/files#diff-7a7786791dd24256a89805479916cfddR8
and
https://github.com/marionettejs/marionettejs.com/pull/101/files#diff-b8b2c04c1ffbd5a7760c721840d38af1R131

As for the prev/next arrows, you could go ahead and inline svg them.  Even if we end up reusing that arrow, inlined it's soooo small:

```sass
.prev {
      left: 0;
      background: $mint url('data:image/svg+xml;utf8,<svg xmlns="http://www.w3.org/2000/svg" width="18" height="18" viewBox="0 0 768 768"><path color="currentColor" d="M493.28 237.28l-45.28-45.28-192 192 192 192 45.28-45.28-146.72-146.72z"></path></svg>') no-repeat 45% center;
    }

    .next {
      right: 0;
      background: $mint url('data:image/svg+xml;utf8,<svg xmlns="http://www.w3.org/2000/svg" width="18" height="18" viewBox="0 0 768 768"><path color="currentColor" d="M320 192l-45.28 45.28 146.72 146.72-146.72 146.72 45.28 45.28 192-192z"></path></svg>') no-repeat 55% center;
    }
```

File: src/js/script.js [hunk 1]
```diff
@@ -1,11 +1,57 @@
...
+    //     pager: '.slider2',
+    //     pagerTemplate: '<span></span>',
+    //     paused: true
+    // });
+
+    // CSS3 Slider - Videos Section
+    // ---
+
+    (function() {
+        var $slideshowWrap = $('.video_slideshow_scroller'),
```
Comment (samccone) [inline src/js/script.js]: gosh at some point we really should use marionette ha

Comment (samccone) [inline src/js/script.js]: another refactor tho

File: src/js/script.js [hunk 2]
```diff
@@ -1,11 +1,57 @@
...
+            $slider = $slideshowWrap.children('.video_slideshow'),
+            $slide = $slider.children().first(),
+            slideCount = $slider.children().length,
+            pos = 0;
+
+        function setTransform() {
+            var distance = (-pos * ($slide.width() + parseInt($slide.css('margin-right').replace(/px/g, ''), 10)));
+            $slider.css('transform', 'translate3d(' + distance + 'px,0,0)');
+
+            if (!pos) {
```
Comment (samccone) [inline src/js/script.js]: `$slideshowPrev[pos ? 'show' : 'hide']()`

File: src/js/script.js [hunk 3]
```diff
@@ -1,11 +1,57 @@
...
+                $slideshowPrev.hide();
+            } else {
+                $slideshowPrev.show();
+            }
+
+            if (pos === slideCount - 1) {
```
Comment (samccone) [inline src/js/script.js]: ditto ^

**Comment by samccone** [comment]
we will want to drop the script tag for cycle also right?

**Comment by kkemple** [comment]
yeah all good, i was in a bit of rush to get to work, just wanted to get PR up before someone else did the work also. I will clean and refactor this afternoon. Let me know if you think of anything else!

File: src/js/script.js (hunk 2 above)
Comment (paulfalgout) [inline src/js/script.js]: alternatively `$slideshowPrev.toggle(pos);`

File: src/js/script.js (hunk 1 above)
Comment (kkemple) [inline src/js/script.js]: yeah we really should dog food the site, at least the homepage

Comment (kkemple) [inline src/js/script.js]: the slider screams collection/composite view lol

**Comment by kkemple** [comment]
@samccone @paulfalgout okay so i updated the arrows to use inline svgs, removed the pngs, refactored the js around showing and hiding the buttons, removed script tag for cycle slider, and removed commented out initialization for cycle slider

:shipit:

**Comment by samccone** [comment]
sweeeet just testingreal quick

File: src/js/script.js [hunk 4]
```diff
@@ -1,11 +1,40 @@
...
+
+        function slideshowNext() {
+            pos = (pos === slideCount - 1) ? pos : pos + 1;
+            setTransform();
+        };
+
+        $(window).on('resize', _.throttle(setTransform, 300));
+        $slideshowNext.on('click', slideshowNext);
+        $slideshowPrev.on('click', slideshowPrev);
+    })();
```
Comment (jasonLaster) [inline src/js/script.js]: Good code!

**Comment by samccone** [comment]
amazing

File: src/js/script.js (hunk 4 above)
Comment (kkemple) [inline src/js/script.js]: wow thanks! :blush:

**Comment by jmeas** [comment]
Nice work @kkemple!

**Comment by kkemple** [comment]
glad to help out! :smile:

**Pull request merged**